soh/soh/Enhancements/randomizer/3drando/location_access/locacc_dodongos_cavern.cpp

```diff
@@ -235,7 +235,7 @@ void AreaTable_Init_DodongosCavern() {
   areaTable[DODONGOS_CAVERN_BOSS_ROOM] = Area("Dodongos Cavern Boss Room", "Dodongos Cavern", DODONGOS_CAVERN, NO_DAY_NIGHT_CYCLE, {
                   //Events
                   EventAccess(&DodongosCavernClear, {[]{return DodongosCavernClear || (Here(DODONGOS_CAVERN_BOSS_ROOM, []{return HasExplosives || (CanUse(MEGATON_HAMMER) && CanShield);}) && (Bombs || GoronBracelet) && (IsAdult || Sticks || KokiriSword));},
-                                         /*Glitched*/[]{return Here(DODONGOS_CAVERN_BOSS_ROOM, []{return HasExplosives || (CanUse(MEGATON_HAMMER) && CanShield) || (GlitchBlueFireWall && BlueFire);}) && (HasExplosives || GoronBracelet) && (IsAdult || Sticks || KokiriSword || CanUse(MEGATON_HAMMER));}}),
+                                         /*Glitched*/[]{return Here(DODONGOS_CAVERN_BOSS_ROOM, []{return HasExplosives || (CanUse(MEGATON_HAMMER) && CanShield) || (GlitchBlueFireWall && BlueFire && HasBottle && BlueFireAccess);}) && (HasExplosives || GoronBracelet) && (IsAdult || Sticks || KokiriSword || CanUse(MEGATON_HAMMER));}}),
```

Comment:
Just making sure I'm following the logic, this is here because the Blue Fire Arrows don't work for the bombable floor above KD right? But the bottle of blue fire does still work? Is it that way in N64 rando as well?

Reply:
Looks like you explained in your commit message. Which way I swing on this one is gonna depend on what n64 rando does. If they also have their logic set up this way then we can keep that as is. If not, I'd rather us take the time to figure out how to make the collision on the bombable floor work better.

Reply:
Just checked, if I'm reading their Glitched World JSON files right, they don't consider having Blue Fire as being able to reach King Dodongo, so we're all good! Disregard these comments.